js/modules/k6/http/request.go

```diff
@@ -444,7 +454,11 @@ func (h *HTTP) Batch(ctx context.Context, reqsV goja.Value) (goja.Value, error)
 	}
 
 	if err != nil {
-		return nil, err
+		if state.Options.Throw.Bool {
+			return nil, err
+		}
+		state.Logger.WithField("error", err).Warn("A batch request failed")
+		return rt.ToValue(results), nil
```

Comment:
Reading [ToValue](https://pkg.go.dev/github.com/dop251/goja?utm_source=godoc#Runtime.ToValue) documentation has the following paragraph(near the very end, a few pages down):
```
Any other Go function is wrapped so that the arguments are automatically converted into the required Go types and the return value is converted to a JavaScript value (using this method). If conversion is not possible, a TypeError is thrown. 
```

Which to me means that most of our usage of `rt.ToValue` is probably not necessary, including this one.

Reply:
@MStoykov WDYT about https://github.com/k6io/k6/pull/2045#discussion_r646379357?

Reply:
Actually looking at `common.Bind` where we actually return `func(call goja.FunctionCall) goja.Value` and have to `ToValue` we apparently call it regardless ... so again even less needed. But Goja would've called it, either way, it's just that in this case we wrap this particular function call ... instead of goja doing it :facepalm: 
https://github.com/k6io/k6/blob/da5eebc638d9917cedfe98e232a7eed4cb69a157/js/common/bridge.go#L246-L253

Also the part about multiple return values will not work ... and you can very easily make k6 panic if you write the wrong go code like `func SomeThingThatGetsBinded() (string, string)` ....

Reply:
Thanks for confirming this. I've seen both usages used, but it's good to know we don't have to convert it beforehand.

Resolved by 6b8cd309.